docs/versions/latest/Release.md

```diff
@@ -1,33 +1,38 @@
-[//]: # (VERSION_PLACEHOLDER DO NOT DELETE)
-[//]: # (Used when working on a new release. Placed together with the Version.md)
-[//]: # (Nothing here is optional. If a step must not be performed, it must be said so)
-[//]: # (Do not fill the version, it will be done automatically)
-[//]: # (Quick Intro to what is the focus of this release)
+[//]: # "VERSION_PLACEHOLDER DO NOT DELETE"
```

Comment:
Hey, not sure why you changed this, but we should revert this

Reply:
Hi. 
Sure. Will re-do. It's probably the prettier made a changes automatically on a save of the file.